client/src/modules/stock/inventory-adjustment/inventory-adjustment.js

```diff
@@ -0,0 +1,273 @@
+angular.module('bhima.controllers')
+  .controller('StockInventoryAdjustmentController', StockInventoryAdjustmentController);
+
+// dependencies injections
+StockInventoryAdjustmentController.$inject = [
+  'NotifyService', 'SessionService', 'util',
+  'bhConstants', 'ReceiptModal', 'StockFormService', 'StockService',
+  'uiGridConstants',
+];
+
+/**
+ * @class StockInventoryAdjustmentController
+ *
+ * @description
+ * This module exists to make sure that existing stock can be adjusted entirely
+ */
+function StockInventoryAdjustmentController(
+  Notify, Session, util, bhConstants, ReceiptModal, StockForm,
+  Stock, uiGridConstants,
+) {
+  const vm = this;
+
+  // global variables
+  vm.Stock = new StockForm('StockInventoryAdjustment');
+  vm.movement = {};
+  vm.ROW_ERROR_FLAG = bhConstants.grid.ROW_ERROR_FLAG;
+
+  vm.onDateChange = date => {
+    vm.movement.date = date;
+  };
+
+  vm.onChangeDepot = depot => {
+    vm.depot = depot;
+    loadInventories(vm.depot);
+  };
+
+  // bind constants
+  vm.enterprise = Session.enterprise;
+  vm.maxLength = util.maxLength;
+  vm.maxDate = new Date();
+
+  // bind methods
+  vm.addItems = addItems;
+  vm.removeItem = removeItem;
+  vm.configureItem = configureItem;
+  vm.checkValidity = checkValidity;
+  vm.submit = submit;
+  vm.selectedLots = [];
+  vm.onLotSelect = onLotSelect;
+
+  // grid columns
+  const columns = [
+    {
+      field : 'status',
+      width : 25,
+      displayName : '',
+      cellTemplate : 'modules/stock/exit/templates/status.tmpl.html',
+      enableFiltering : false,
+    },
+
+    {
+      field : 'code',
+      width : 120,
+      displayName : 'TABLE.COLUMNS.CODE',
+      headerCellFilter : 'translate',
+      cellTemplate : 'modules/stock/exit/templates/code.tmpl.html',
+    },
+
+    {
+      field : 'description',
+      displayName : 'TABLE.COLUMNS.DESCRIPTION',
+      headerCellFilter : 'translate',
+      cellTemplate : 'modules/stock/exit/templates/description.tmpl.html',
+      enableSorting : true,
+    },
+
+    {
+      field : 'label',
+      width : 150,
+      displayName : 'TABLE.COLUMNS.LOT',
+      headerCellFilter : 'translate',
+      cellTemplate : 'modules/stock/inventory-adjustment/templates/lot.tmpl.html',
+      enableSorting : true,
+    },
+
+    {
+      field : 'available_lot',
+      width : 150,
+      displayName : 'INVENTORY_ADJUSTMENT.OLD_QUANTITY',
+      headerCellFilter : 'translate',
+      cellTemplate : 'modules/stock/exit/templates/available.tmpl.html',
+    },
+
+    {
+      field : 'quantity',
+      width : 180,
+      displayName : 'INVENTORY_ADJUSTMENT.NEW_QUANTITY',
+      headerCellFilter : 'translate',
+      cellTemplate : 'modules/stock/inventory-adjustment/templates/quantity.tmpl.html',
+      aggregationType : uiGridConstants.aggregationTypes.sum,
```

Comment:
I don't think we should be filtering on these columns:
![image](https://user-images.githubusercontent.com/896472/80528886-8e100f80-898e-11ea-8843-6e843a55de25.png)